### ISSUE TITLE
"OverflowError: timeout is too large" from CookieJar._expire_cookie

### ISSUE THREAD

**Issue by lmacken** [aio-libs/aiohttp#967]
After upgrading from 0.21.6 to 0.22 my code crashes with this exception:

```
  File "/opt/rh/rh-python35/root/usr/lib64/python3.5/asyncio/base_events.py", line 295, in run_
forever                                                                                        
    self._run_once()                                                                           
  File "/opt/rh/rh-python35/root/usr/lib64/python3.5/asyncio/base_events.py", line 1218, in _ru
n_once                                                                                         
    event_list = self._selector.select(timeout)                                                
  File "/opt/rh/rh-python35/root/usr/lib64/python3.5/selectors.py", line 432, in select        
    fd_event_list = self._epoll.poll(timeout, max_ev)                                          
OverflowError: timeout is too large 
```

After some debugging, I discovered that it's this task that is triggering it:

```
<TimerHandle when=1500104826.0 CookieJar._expire_cookie('visid_incap_116519') created at /home/luke/py35env/lib64/python3.5/site-packages/aiohttp/helpers.py:650>
```

Environment:

```
Linux 3.10.0-327.22.2.el7.x86_64
Python 3.5.1
```

**Comment by asvetlov** [comment]
Thanks you for catching regression.
Will publish a fix soon.

**Comment by asvetlov** [comment]
Bugfix has been published in aiohttp==0.22.1
